src/styles/shiki.scss

```diff
@@ -82,29 +89,34 @@ pre {
 
 pre .code-container {
 	/* Give it some space to breathe */
-	padding: 12px;
+	padding: var(--shiki_padding) 0;
+	display: flex;
+
+	code {
+		flex-grow: 1;
+	}
 }
+
 pre.shiki {
 	overflow-x: auto;
 }
-pre.shiki:hover .dim {
-	opacity: 1;
-}
-pre.shiki div.dim {
-	opacity: 0.5;
-}
-pre.shiki div.dim,
-pre.shiki div.highlight {
-	margin: 0;
-	padding: 0;
+pre.shiki div.line {
+	// must be the same value as used for line-height
+	min-height: 1.7em;
+	padding: 0 var(--shiki_padding);
+	border-top: 1px solid transparent;
```

Comment:
How does a 2px border look to you all? Do you think it'd help with scannability?

Reply:
Yep, that's a huge improvement IMO - I'll do that :)